src/uct/ib/rdmacm/rdmacm_cm.h

```diff
@@ -77,6 +77,7 @@ typedef struct uct_rdmacm_cm_device_context {
     uint32_t        log_reserved_qpn_granularity;
     uint32_t        num_dummy_qps;
     struct ibv_cq   *cq;
+    uint8_t         is_eth;
```

Comment:
Maybe add `bitmap` to the var name, and/or use `ucs_bitmap_t`, to emphasize that it is a bitmap and not a number.